.ci/bump-stack-version.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# Given the stack version this script will bump the version.
+#
+# This script is executed by the automation we are putting in place
+# and it requires the git add/commit commands.
+#
+set -euo pipefail
+MSG="parameter missing."
+VERSION=${1:?$MSG}
+
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+
+if [ "${OS}" == "darwin" ] ; then
+	SED="sed -i .bck"
+else
+	SED="sed -i"
+fi
+
+echo "Update stack with version ${VERSION}"
+${SED} -E -e "s#(image: docker\.elastic\.co/.*):[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1:${VERSION}#g" testing/environments/snapshot-oss.yml
+${SED} -E -e "s#(image: docker\.elastic\.co/.*):[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1:${VERSION}#g" testing/environments/snapshot.yml
+
+echo "Commit changes"
+git config user.email
+git checkout -b "update-stack-version-$(date "+%Y%m%d%H%M%S")"
```

Comment:
In order to reuse an existing PR then the branch creating should only happen if set.

In other words,

There is a requirement to support:
1) Reusing an existing PR if there is already an open PR with a previous bump.
2) Creating a new PR if the above is not true or it's configured to create always independently.